### PR TITLE
Add ResellerClub pricing

### PR DESCRIPTION
**Pull request by nbebout**
Add ResellerClub pricing support

### CODE REVIEW

File: classes/ResellerClubClient.php [hunk 1]
```diff
@@ -213,6 +213,73 @@ private function pricingType(string $type) : int {
...
+            $donutsgroup3 = ['apartments', 'bingo', 'capital', 'careers', 'claims', 'clinic', 'coach', 'codes', 'condos', 'coupons', 'cruises', 'dating', 'delivery', 'dental', 'diamonds', 'engineering', 'expert', 'finance', 'financial', 'flights', 'fund', 'furniture', 'golf', 'healthcare', 'hockey', 'holdings', 'holiday', 'insure', 'jewelry', 'lease', 'legal', 'limo', 'maison', 'memorial', 'partners', 'pizza', 'recipes', 'restaurant', 'surgery', 'tax', 'taxi', 'tennis', 'theater', 'tienda', 'tours', 'university', 'ventures', 'viajes', 'villas', 'voyage'];
+            $dom = ['biz', 'info', 'org', 'us'];
+            if (in_array($tld, $donutsgroup1)) { return 'donutsgroup1'; }
+            else if (in_array($tld, $donutsgroup2)) { return 'donutsgroup2'; }
+            else if (in_array($tld, $donutsgroup3)) { return 'donutsgroup3'; }
+            else if (in_array($tld, $dom)) { return "dom$tld"; }
+            else if ($tld == 'com') { return 'domcno'; }
+            else { return "dot$tld"; }
```
Comment (lfkeitel) [inline classes/ResellerClubClient.php]: This else isn't needed as every other branch will return if a condition is satisfied. Simply remove the else and braces and leave the return statement.

File: classes/EnomClient.php [hunk 2]
```diff
@@ -199,4 +199,12 @@ public function GetResellerPrice(string $tld) : array {
  
         return $prices;
     }
+
+    public function GetAllPrices(array $tldarray) : array {
```
Comment (lfkeitel) [inline classes/EnomClient.php]: Add this new function to the RegistrarClient interface.

**Comment by nbebout** [comment]
I updated it.  Do I have to tell you I updated a PR, or does it automatically notify you?

**Comment by lfkeitel** [comment]
I get notifications that it's been updated since you have me as a reviewer.

**Pull request merged**